### ISSUE TITLE
Three.js examples have GL error on Pixel 1

### ISSUE THREAD

**Issue by jdm** [servo/servo#22082]
When I load https://threejs.org/examples/webgl_animation_cloth.html on my pixel device, I see:
```
11-01 15:18:31.409 18451 18561 D simpleservo: THREE.WebGLRenderer
11-01 15:18:31.409 18451 18561 D simpleservo: 98
11-01 15:18:31.418 18451 18576 E simpleservo: Last GL operation failed: GetShaderPrecisionFormat(35633, 36338, WebGLSender(..))
11-01 15:18:31.418 18451 18561 D simpleservo: thread 'WebGLThread' panicked at 'assertion failed: `(left == right)`
11-01 15:18:31.418 18451 18561 D simpleservo:   left: `1280`,
11-01 15:18:31.418 18451 18561 D simpleservo:  right: `0`: Unexpected WebGL error: 0x500 (1280)', components/canvas/webgl_thread.rs:1231:9
11-01 15:18:31.418 18451 18561 D simpleservo: note: Run with `RUST_BACKTRACE=1` for a backtrace.
```
These values correspond to:
GetShaderPrecisionFormat(constants::VERTEX_SHADER, constants::HIGH_FLOAT), and the error is constants::INVALID_ENUM.

**Comment by CYBAI** [comment]
Should this issue be closed after `gleam` is upgraded to `0.6.5` in Servo instead?

**Comment by jdm** [comment]
Yes.
